### PR TITLE
Menu buttons

### LINKED ISSUE #7878: Menu buttons on mobile
image
lets change it to 
image

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [X] Feature
- [ ] Refactoring

## Context

- [X] Closes #7878
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [X] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot from 2023-11-01 00-29-16

### <samp>🤖 Generated by Copilot at a2d250c</samp>

Updated the `MobileFilter` component to use flat buttons for a better UI. Added a `flat` prop and class to the `NeoButton` component to enable borderless buttons.

### <samp>🤖 Generated by Copilot at a2d250c</samp>

> _We're coding up a storm, me hearties, on the `MobileFilter` screen_
> _We're making it look nice and neat, with buttons flat and clean_
> _We've added a new prop to the `NeoButton` style_
> _So we can switch the border off and sail the seas with style_

### CODE REVIEW

**Review by preschian — APPROVED** [review]

**Comment by daiagi** [comment]
@exezbcz  how should the "Reset All" button look in hover?

File: libs/ui/src/components/NeoButton/NeoButton.vue [hunk 1]
```diff
@@ -39,6 +40,7 @@ withDefaults(
     active?: boolean
     fixedWidth?: boolean
     noShadow?: boolean
+    flat?: boolean
```
Comment (daiagi) [inline libs/ui/src/components/NeoButton/NeoButton.vue:43]: 1. depending on the required looks in hover, you may be able to use `text` variant
2. if not - then plz change the prop name to `noBorder`

**Review by daiagi — CHANGES_REQUESTED** [review]

**Comment by exezbcz** [comment]
@daiagi same as hover over links - as in footer

-> change to grey

**Comment by daiagi** [comment]
> @daiagi same as hover over links - as in footer
> 
> -> change to grey

@pbkompasz 
no need for a new prop then. use `text` variant

**Review by daiagi — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 4.56 USD/DOT ~ 4.386 $DOT
🧗 [16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt](https://polkadot.subscan.io/account/16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt)
🔗 [0xd1903ee41cb47020e093e2d9ff48f69f339290ae92097c0e03b47f34b6f1ba60](https://polkadot.subscan.io/extrinsic/0xd1903ee41cb47020e093e2d9ff48f69f339290ae92097c0e03b47f34b6f1ba60)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**